src/Components/Navbar/Components/NavbarMenu/NavbarMenu.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import NavbarSubMenu from './NavbarSubMenu';
+import '../NavbarMenu/NavbarMenu.scss';
+class NavbarMenu extends Component {
+  onFindId = (id) => {
+    this.props.updateSubMenu(id);
+  };
+
+  render() {
+    const { onSubMenuList } = this.props;
+    return (
+      <div className='NavbarMenu'>
+        <ul className='gnbMenu'>
+          {onSubMenuList.map((item) => {
+            return (
+              <li
+                key={item.id}
+                className={`${
+                  item.menuName === '나의 라코스테 폴로' ? 'hoverNot' : ''
+                } ${item.menuName === 'SALE' ? 'myPolo' : ''}`}
```

Comment:
```suggestion
                className={`${
                  item.menuName === '나의 라코스테 폴로' ? 'hoverNot' : item.menuName === 'SALE' ? 'myPolo' : ''}`}
```
이렇게 쓰시거나
```suggestion
                const { menuName } = item
                const menuNameClass = menuName === '나의 라코스테 폴로' ? 'hoverNot' : menuName === 'SALE' ? 'myPolo' : ''
                                
                className={`${menuNameClass}`}
```
이렇게 쓰면 어떨까요?

Reply:
리뷰 감사합니다 수정후 바로 올리겟습니다.